apps/frontend/src/app/3_organisms/GettingStartedPopup/GettingStartedPopup.tsx

```diff
@@ -120,11 +124,31 @@ export const GettingStartedPopup: FC<GettingStartedPopupProps> = ({
                 >
                   {item.description}
                 </Heading>
-                <Link
-                  text={t(translations.gettingStartedPopup.buttons.learnMore)}
-                  href={item.href}
-                  openNewTab
-                />
+                {Array.isArray(item.href) ? (
+                  <>
+                    {item.href.map((href, index) => (
+                      <Link
+                        text={
+                          item.learnMore
+                            ? item.learnMore[index]
+                            : t(
+                                translations.gettingStartedPopup.buttons
+                                  .learnMore,
+                              )
+                        }
+                        href={href}
+                        openNewTab
```

Comment:
```suggestion
```
You don't need to provide it since it's an optional prop and its default value is `true`.